desktop/renderer-app/src/stores/class-room-store.ts

```diff
@@ -443,6 +450,22 @@ export class ClassRoomStore {
         this.addMessage(RTMessageType.ChannelMessage, text, this.userUUID);
     };
 
+    public onUserGuide = (): void => {
+        if (
+            this.messages.length > 0 &&
+            this.messages[this.messages.length - 1].type === RTMessageType.UserGuide
```

Comment:
why? add comment

Reply:
Why not use: `this.message.some(({ type }) => type === RTMessageType.UserGuide)`

Why do I need to take the last digit of the array?(`this.messages[this.messages.length - 1]`)

Reply:
when rejoin or refresh classroom that the user guide message always at the end

Reply:
Ok. Then it would be better to add this sentence to the comment.